bugbug/issue_snapshot.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+def rollback(issue, when=None):
+    assert when is None
```

Comment:
Nit: add an assertion message that says rollbacking to a specific point in history is not supported yet.